### DISCUSSION TITLE
aerial vs. symbols-outline

### DISCUSSION THREAD

**Discussion by jpfender** [LazyVim/LazyVim#1766] [Q&A] [answered]
I noticed that the `editor.aerial` and `editor.symbols-outline` extras both use the same mapping by default, `<leader>cs`. They also pretty much cover the same functionality. What is everyone's preference between the two plugins? @folke which one do you use?

**Comment by cStralpt** [comment]
same confusion here, i wonder why @folke  add that as extras, there's absolutely a good reason i think.

**Comment by folke** [comment, marked as answer]
I configured both to look pretty much the same. Aerial has the benefit that it also has a lualine component which again I made look the same as `nvim-navic`.

So aerial = navic + symbols-outline

**Reply by lkhphuc** [reply]
aerial = navic +symbols-outline + [navbuddy](https://github.com/SmiteshP/nvim-navbuddy) as well.
Though it seems less well-polished, for lesser use functionalities, 1-for-3 it's a win.

**Reply by Demianeen** [reply]
I don't see the custom keymaps like `nvim-navbuddy` has. Does Aerial provide any additional keymaps except navigation?

**Comment by pkazmier** [comment]
As another option, you don't need aerial or symbols-outline if you use the _experimental_ symbols outline in Neo-tree as documented in #1203. For breadcrumbs, you'll need to install the navic extra. I like this setup as the functionality for a symbol outline is already included in Neo-tree, so I don't need to install either aerial or symbols-outline.
